anndata2rdf/src/curated_data/Curated_Lung_Data_KG_V1 - Sheet1.csv

```diff
@@ -0,0 +1,4 @@
+Dataset (indv datasets within larger group),Full name dataset (top of page),CxG Link,h5ad link,Reference_DOI,Bionetworks reference,Standard category present? (T/F),Standard category cell_type present? (T/F),Author Category Cell Type Field Name,Content,Value type(s),Notes,Study Short Name,CxG Dataset Collection X,Is the dataset Normal or Normal/Diseased,Stage
+Human CellCards Multi-Study CellRef 1.0 Atlas,Human CellCards Multi-Study CellRef 1.0 Atlas,https://cellxgene.cziscience.com/e/eb499fd8-7000-419f-8854-926b9b61b11f.cxg/,https://datasets.cellxgene.cziscience.com/443f7fb8-2a27-47c3-98f6-6a603c7a294e.h5ad,DOI: 10.1038/s41467-023-40173-5,T,T,T,"celltype_lev
```

Comment:
We were using following fields as author cell types;

- celltype_level1
- celltype_level2
- celltype_level3
- celltype_level3_fullname
- lineage_level1
- lineage_level2

Could you please add them as well